src/main.rs

```diff
@@ -44,6 +44,9 @@ enum Command {
     /// Get the address of an acccount from account index
     Account {
         account_index: usize,
+        #[clap(long)]
+        export: bool,
```

Comment:
Perhaps we should have a separate command for this, just like we do for `import`? If so, we should also ensure the command is well documented.

Happy to approve for now so that it's at least possible to export in the meantime :+1: